packages/web/.env.example

```
@@ -1,5 +1,22 @@
-# Replace with your DB credentials!
-DATABASE_URL="libsql://db-name-user.turso.io"
+# Database (Turso/LibSQL)
+DATABASE_URL="http://localhost:8080"
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)